thamos/config.py

```diff
@@ -329,6 +335,12 @@ def create_default_config(
         platform = discover_platform()
         base_image = discover_base_image()
         base_image = base_image if base_image is not None else "null"
+        cudnn_version = discover_cudnn_version()
```

Comment:
These can all be changed simply to
```suggestion
        cudnn_version = discover_cudnn_version() or "null"
```